app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotDaoIT.java

```diff
@@ -53,28 +53,30 @@ void populateDatabaseTables() {
         populator.setScripts(
                 new ClassPathResource("fixtures/experiment.sql"),
                 new ClassPathResource("fixtures/scxa_analytics.sql"),
-                new ClassPathResource("fixtures/scxa_coords.sql"),
                 new ClassPathResource("fixtures/scxa_cell_group.sql"),
-                new ClassPathResource("fixtures/scxa_cell_group_membership.sql"));
+                new ClassPathResource("fixtures/scxa_cell_group_membership.sql"),
+                new ClassPathResource("fixtures/scxa_dimension_reduction.sql"),
+                new ClassPathResource("fixtures/scxa_coords.sql"));
         populator.execute(dataSource);
     }
 
     @AfterAll
     void cleanDatabaseTables() {
         populator.setScripts(
-                new ClassPathResource("fixtures/scxa_cell_group_marker_gene_stats-delete.sql"),
-                new ClassPathResource("fixtures/scxa_cell_group_marker_genes-delete.sql"),
+                new ClassPathResource("fixtures/scxa_coords-delete.sql"),
+                new ClassPathResource("fixtures/scxa_dimension_reduction-delete.sql"),
                 new ClassPathResource("fixtures/scxa_cell_group_membership-delete.sql"),
                 new ClassPathResource("fixtures/scxa_cell_group-delete.sql"),
-                new ClassPathResource("fixtures/scxa_coords-delete.sql"),
                 new ClassPathResource("fixtures/scxa_analytics-delete.sql"),
                 new ClassPathResource("fixtures/experiment-delete.sql"));
         populator.execute(dataSource);
     }
 
+
     @Test
     void fetchCellPlotWithK() {
-        assertThat(subject.fetchCellPlotWithK("E-ENAD-53", 38, "umap", Map.of("n_neighbors",50)))
+        assertThat(
+                subject.fetchCellPlotWithK("E-ENAD-53", 14, "umap", Map.of("n_neighbors", 15)))
```

Comment:
Maybe in the next iteration, will randomise this hardcoded experiment accession.

Reply:
The problem is that we have a subset of the rows for each experiment accession, and I’m not sure how feasible that would be.

Reply:
Ok, thanks.